composer.json

```
@@ -18,11 +19,13 @@
         "sylius/resource-bundle": "^1.6",
         "symfony/config": "^5.4 || ^6.4 || ^7.0",
         "symfony/dependency-injection": "^5.4 || ^6.4 || ^7.0",
+        "symfony/doctrine-bridge": "^5.4 || ^6.4 || ^7.0",
```

```suggestion
        "symfony/doctrine-bridge": "^5.4 || ^6.3 || ^7.0",
```

`symfony/doctrine-bridge` v6.4 removes `doctrine/persistence` v2 from its deps, and this has a lot of implications for Sylius related packages